x-pack/plugins/fleet/server/services/epm/packages/get.test.ts

```diff
@@ -0,0 +1,171 @@
+            created_by: 'system',
+            updated_at: '2020-12-21T19:22:04.902Z',
+            updated_by: 'system',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-21T19:22:04.905Z',
+          version: 'WzIxNSwxXQ==',
+          score: 0,
+        },
+        {
+          type: 'ingest-package-policies',
+          id: 'dcf83172-c38e-4501-b236-9f479da8a7d6',
+          attributes: {
+            name: 'system-2',
+            description: '',
+            namespace: 'default',
+            policy_id: '11111-111111-11111-11111',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
+        {
+          type: 'ingest-package-policies',
+          id: 'dcf83172-c38e-4501-b236-9f479da8a7d6',
+          attributes: {
+            name: 'system-3',
+            description: '',
+            namespace: 'default',
+            policy_id: '22222-22222-2222-2222',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
+        {
+          type: 'ingest-package-policies',
+          id: 'dcf83172-c38e-4501-b236-9f479da8a7d6',
+          attributes: {
+            name: 'system-4',
+            description: '',
+            namespace: 'default',
+            policy_id: '33333-33333-333333-333333',
+            enabled: true,
+            output_id: '',
+            inputs: [],
+            package: { name: 'system', title: 'System', version: '0.10.4' },
+            revision: 1,
+            created_at: '2020-12-22T21:28:05.380Z',
+            created_by: 'elastic',
+            updated_at: '2020-12-22T21:28:05.380Z',
+            updated_by: 'elastic',
+          },
+          references: [],
+          migrationVersion: { 'ingest-package-policies': '7.11.0' },
+          updated_at: '2020-12-22T21:28:05.383Z',
+          version: 'WzE1NTAsMV0=',
+          score: 0,
+        },
```

Comment:
Is this much mocking and setup really necessary for testing paging?

Reply:
yes and no. Some packge policies (integration policies) can have the same  Agent Policy id (with the exception of Endpoint, every other integration can be added to agent policies multiple times)